src/config.c

```diff
@@ -1071,15 +1067,19 @@ int config_write_template(const char *file, const struct config *cfg)
 
 	(void)re_fprintf(f,
 			 "\n# HTTP request module\n"
-			 "# httpreq_ca\t\ttrusted1.pem\n"
-			 "# httpreq_ca\t\ttrusted2.pem\n"
-			 "# httpreq_dns\t\t1.1.1.1\n"
-			 "# httpreq_dns\t\t8.8.8.8\n"
-			 "# httpreq_hostname\tmyserver\n"
-			 "# httpreq_cert\t\tcert.pem\n"
-			 "# httpreq_key\t\tkey.pem\n"
-			 );
-
+			 "#httpreq_ca\t\ttrusted1.pem\n"
+			 "#httpreq_ca\t\ttrusted2.pem\n"
+			 "#httpreq_dns\t\t1.1.1.1\n"
+			 "#httpreq_dns\t\t8.8.8.8\n"
+			 "#httpreq_hostname\tmyserver\n"
+			 "#httpreq_cert\t\tcert.pem\n"
+			 "#httpreq_key\t\tkey.pem\n");
```

Comment:
Placing multicast config at the end of the template is ok. But here is an unrelated change of the httpreq config. At least this should be two commits. Better two PRs.

Reply:
undid change